docs/guides/options.md

```diff
@@ -1,138 +1,312 @@
+
+### `height`
+
+> Type: `String|Number`
+
+Sets the display height of the video player in pixels.
+
+### `loop`
+
+> Type: `Boolean`
+
+Causes the video to start over as soon as it ends.
+
+### `muted`
+
+> Type: `Boolean`
+
+Will silence any audio by default.
+
+### `poster`
+
+> Type: `String`
+
+A URL to an image that displays before the video begins playing. This is often a frame of the video or a custom title screen. As soon as the user hits "play" the image will go away.
+
+### `preload`
+
+> Type: `String`
+
+Suggests to the browser whether or not the video data should begin downloading as soon as the `<video>` element is loaded. Supported values are:
+
+- `'auto'`
+
+  Start loading the video immediately (if the browser supports it). Some mobile devices will not preload the video in order to protect their users' bandwidth/data usage. This is why the value is called 'auto' and not something more conclusive like `'true'`.
+
+  _This tends to be the most common and recommended value as it allows the browser to choose the best behavior._
+
+- `'metadata'`
+
+  Load only the meta data of the video, which includes information like the duration and dimensions of the video.
+
+- `'none'`
+
+  Don't preload any data. The browser will wait until the user hits "play" to begin downloading.
+
+### `src`
+
+> Type: `String`
+
+The source URL to a video source to embed.
+
+### `width`
+
+> Type: `String|Number`
+
+Sets the display height of the video player in pixels.
+
+## Video.js-specific Options
+
+Each option is `undefined` by default unless otherwise specified.
+
+### `aspectRatio`
+
+> Type: `String`
+
+Puts the player in [fluid](#fluid) mode and the value is used when calculating the dynamic size of the player. The value should represent a ratio - two numbers separated by a colon (e.g. `"16:9"` or `"4:3"`).
+
+### `children`
+
+> Type: `Array`
+
+### `fluid`
+
+> Type: `Boolean`
+
+When `true`, the Video.js player will have a fluid size. In other words, it will scale to fit its container.
 
-```html
-<video data-setup='{ "controls": true, "autoplay": false, "preload": "auto" }'...>
+Also, if the `<video>` element has the `"vjs-fluid"`, this option is automatically set to `true`.
+
+### `inactivityTimeout`
+
+> Type: `Number`
+
+Video.js indicates that the user is interacting with the player by way of the `"vjs-user-active"` and `"vjs-user-inactive"` classes and the `"useractive"` event.
+
+The `inactivityTimeout` determines how many milliseconds of inactivity is required before declaring the user inactive. A value of `0` indicates that there is no `inactivityTimeout` and the user will never be considered inactive.
+
+### `language`
+
+> Type: `String`
+
+A [language code][lang-codes] matching one of the available languages in the player. This sets the initial language for a player, but it can always be changed.
```

Comment:
I think that this defaults to english, especially if there is no translation available in the language that you want to use.